bin/bazel_to_go.py

```diff
@@ -127,20 +132,26 @@ def bazel_to_vendor(WKSPC):
         print "WORKSPACE file not found in " + WKSPC
         print "prog BAZEL_WORKSPACE_DIR"
         return -1
+    mixer_adapter_repos = WKSPC + "/adapter_author_deps.bzl"
```

Comment:
agreed that would better, but I didn't want to rewrite the whole thing in this PR.